src/pybind/mgr/dashboard/frontend/src/app/shared/components/bottom-links/bottom-links.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'cd-bottom-links',
+  templateUrl: './bottom-links.component.html',
+  styleUrls: ['./bottom-links.component.scss']
+})
+export class BottomLinksComponent {
+  docTitles: any[] = [
+    { section: 'security', docText: 'Security' },
+    { section: 'help', docText: 'Help' },
```

Comment:
Why not just `text`? 
```suggestion
    { section: 'help', text: 'Help' },
```

Reply:
Done.